org.garudalinux.firedragon.yml

```diff
@@ -0,0 +1,173 @@
+app-id: org.garudalinux.firedragon
+name: FireDragon
+runtime: org.freedesktop.Platform
+runtime-version: 23.08
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: 23.08
+    add-ld-path: .
+command: firedragon
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
```

Comment:
it's either wayland with fallback or x11. NOT both.

Reply:
Fixed in 178c42df127cd855ee5e3cb010cbc0d6cf5ed307.